### ISSUE TITLE
xformer attention does not work

### ISSUE THREAD

**Issue by VladAndronik** [ShivamShrirao/diffusers#91]
### Describe the bug

I compiled the xformer using pip install git+https://github.com/facebookresearch/xformers@1d31a3a#egg=xformers
However, I have not seen any improvement with memory (did not check the speed). It is stated that xformer + gradient checkpointing give 16GB training but I only got ~19-20GB.
How can I check if xformer is enabled? (I received message about xformer not enabled before building it, but message disappeared afterwards). And what is actual improvement it gives?

### System Info

- `diffusers` version: 0.7.0.dev0
- Platform: Linux-5.4.0-126-generic-x86_64-with-glibc2.17
- Python version: 3.8.10
- PyTorch version (GPU?): 1.12.0+cu113 (True)
- Huggingface_hub version: 0.10.1
- Transformers version: 4.23.1
- Using GPU in script?: 3090
- Using distributed or parallel set-up in script?: NO

**Comment by ShivamShrirao** [comment]
How did u install diffusers ?
